ci/conda/recipes/run_conda_build.sh

```diff
@@ -99,9 +99,9 @@ fi
 # Choose default variants
 if hasArg quick; then
    # For quick build, just do most recent version of rapids
-   CONDA_ARGS_ARRAY+=("--variants" "{python: 3.8, rapids_version: 23.02}")
+   CONDA_ARGS_ARRAY+=("--variants" "{python: 3.10, rapids_version: 23.02}")
 else
-   CONDA_ARGS_ARRAY+=("--variants" "{python: 3.8}")
+   CONDA_ARGS_ARRAY+=("--variants" "{python: 3.10}")
```

Comment:
If we remove additional versions of python from the build matrix (`conda_build_config.yaml`), we can remove this flag which specifies that we only want to build with one variant.